src/server/routes/entity/entity.js

```diff
@@ -283,59 +411,52 @@ export function handleDelete(
 	const editorJSON = req.session.passport.user;
 	const {body}: {body: any} = req;
 
-	const entityDeletePromise = bookshelf.transaction((transacting) => {
+	const entityDeletePromise = bookshelf.transaction(async (transacting) => {
+		const otherEntities = await deleteRelationships(orm, transacting, entity);
+
 		const editorUpdatePromise =
 			utils.incrementEditorEditCountById(orm, editorJSON.id, transacting);
```

Comment:
Incrementing the editor count should be taken care of by `saveEntitiesAndFinishRevision`, so I think at the moment you're incrementing it twice.